frontend/resourceadm/components/ConsentPreview/ConsentPreview.test.tsx

```
@@ -4,6 +4,7 @@
 import { textMock } from '@studio/testing/mocks/i18nMock';
 import { testConsentTemplates } from '../../testing/utils/testUtils';
 import { ConsentPreview } from './ConsentPreview';
+import { ValidLanguage } from 'app-shared/types/ResourceAdm';
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)